docker-helper

```diff
@@ -84,6 +84,8 @@ case $CMD in
 
         # if github credentials set, push to github
         if [ -n "${GITHUB_USERNAME:-}" ] && [ -n "${GITHUB_TOKEN:-}" ]; then
+            # GHPR requires an extra image name after the repo
+            # e.g. "chanzuckerberg/<proj_name>/<proj_name>"
```

Comment:
in this example, can you add the full path to the github registry. I think it was something like docker.pkg.github.com or somethign like that